### PR TITLE
kodi{-wayland,-gbm}: use LTS jdk11

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -26838,14 +26838,18 @@ in
...
 
   kodi-gbm = callPackage ../applications/video/kodi {
+    jre_headless = jdk11_headless;
     gbmSupport = true;
   };
 
-  kodi-cli = callPackage ../tools/misc/kodi-cli { };
+  kodi-cli = callPackage ../tools/misc/kodi-cli {
+    jre_headless = jdk11_headless;
+  };
```
Comment (aanderse) [inline pkgs/top-level/all-packages.nix:26852]: `kodi-cli` isn't a variant of `kodi`, it is just a shell script or something. It doesn't take `java` at all so you can remove this.

Comment (petabyteboy) [reply pkgs/top-level/all-packages.nix]: I first tested building it, and it failed, but actually it was a dependency that failed to build. Fixed it now.

**Comment by aanderse** [comment]
@GrahamcOfBorg build kodi-wayland kodi-gbm

**Comment by petabyteboy** [comment]
When applying this please squash and remove the -cli from the commit message

**Comment by aanderse** [comment]
Squash? I only see one commit...

**Comment by petabyteboy** [comment]
Yes, please squash anyways to edit the commit message because I'm too lazy to force push.

**Comment by aanderse** [comment]
Understood. Thanks!

**Pull request merged**